packages/react-components/src/components/TextFieldArea.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react';
+import {State, FieldJson} from '@aemforms/af-core';
+import withRuleEngine from '../utils/HOC';
+import { IHandler } from '../utils/type';
+
+
+const TextFieldArea = (props: State<FieldJson> & IHandler) => {
+   const { id, label, name, value, required, readOnly,  valid, errorMessage, description, minLength, maxLength, visible, enabled, placeholder } = props;
+   const validateState = valid === false ? 'invalid' : 'valid';
+   const error = validateState === 'invalid';
+   const [shortDescription, setShortDescription] = useState(true);
+   const [longDescription, setLongtDescription] = useState(false);
+
+   const handleClick = () => {
+      setShortDescription(!shortDescription);
+      setLongtDescription(!longDescription);
+   };
+  
+
+   const handleChange = (event: React.ChangeEvent<HTMLTextAreaElement>) => {
+      const enteredValue = event.target.value;
+      props.dispatchChange(enteredValue);
+   };
+
+   const handleFocus = () => {
+      props.dispatchFocus();
+   };
+
+   return (
+      <div id={id} className="cmp-adaptiveform-textinput" data-cmp-is="adaptiveFormTextInput"  data-cmp-visible={visible} data-cmp-enabled={enabled}>
+        {label?.visible && <label className="cmp-adaptiveform-textinput__label" htmlFor={id}>{label?.value}</label>}
+        <textarea aria-label={label?.value} className={value ? 'cmp-adaptiveform-textinput__widget cmp-adaptiveform-textinput__widget--filled' : 'cmp-adaptiveform-textinput__widget cmp-adaptiveform-textinput__widget--empty'} name={name} onChange={handleChange} value={value} required={required} readOnly={readOnly} minLength={minLength} maxLength={maxLength} onFocus={handleFocus} placeholder={placeholder} />
```

Comment:
onBlur is not implemented,
Please implement accessibility properly

Reply:
Done!